js/wikiapi.js

```diff
@@ -0,0 +1,30 @@
+import regeneratorRuntime from "regenerator-runtime";
+
+//WikiAPI class for the fetching wiki content. 
+export default class WikiApi {
```

Comment:
Probably doesn't need to be a class.

Reply:
I put it as a class so we can extend functionality later if needed.

Reply:
> if needed

I get that, but we'd implement it that way when we needed it rather than leaving clutter around in the code base. If another developer came up and saw this being a class they'd question where else it was being used.